portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase

```diff
@@ -1107,6 +1107,577 @@ definition {
+
+            ObjectCustomValidation.gotoTab(tabName = "Details");
+
+            Button.clickPublish();
+
+			}
+       
+        task ("When the user creates a new data set") {
+            Navigator.openURL();
+
+            DataSetAdmin.goToDataSetAdminPage();
+
+            DataSetAdmin.createDataSetEntryViaAPI(
+                dataSetName = "Dataset Test",
+                restApplication = "/data-set-manager/fields",
+                restEndpoint = "/",
+                restSchema = "FDSField");
+	
+        }
+
+        task ("And a data set view created via api") {
+            DataSetAdmin.createFDSViewViaAPI(
+                dataSetName = "Dataset Test",
+                dataSetViewDescription = "FDSViewDescription",
+                key_dataSetViewNameList = "View Fields Test");
+
+            DataSetAdmin.goToSpecificViewPage(
+                dataSetName = "Dataset Test",
+                key_viewName = "View Fields Test");
+        }
+
+        task ("When goes to Fields Tab.") {
+            DataSetAdmin.goToTab(tabName = "Filters");
+
+            LexiconEntry.gotoAdd();
+
+            Click(
+                key_fieldName = "Selection",
+                locator1 = "FormViewBuilder#OBJECT_FIELD_BUTTON_DROPDOWN");
+
+            Type(
+                key_text = "Name",
+                locator1 = "TextInput#ANY",
+                value1 = "Transport");
+
+            Click(
+                key_class = "btn-secondary",
+                key_index = 2,
+                locator1 = "Button#ANY_WITH_CLASS_AND_INDEX");
+
+            SelectFrameTop();
+
+            Click.javaScriptClick(
+                key_sortOption = "Relationship_c_fdsViewERC",
+                locator1 = "ObjectCustomViews#SORT_OPTION");
+
+            Select(
+                locator1 = "DataSet#SOURCE_OPTIONS_SELECT",
+                value1 = "Transport");
+        }
+
+        task ("And Preselected Values car,bike,bus") {
+            Click(
+                key_fieldName = "Select a default value for your filter.",
+                locator1 = "ObjectCustomViews#FROM_DATE_INPUT");
+
+            Click(
+                key_sortOption = "car",
+                locator1 = "ObjectCustomViews#SORT_OPTION");
+
+            Click(
+                key_sortOption = "bike",
+                locator1 = "ObjectCustomViews#SORT_OPTION");
+
+            Click(
+                key_sortOption = "bus",
+                locator1 = "ObjectCustomViews#SORT_OPTION");
+        }
+
+        task ("And Select the option Multiple,Include radio button") {
+            Click.clickAtNotVisible(
+                key_radioOption = "Multiple",
+                locator1 = "CommerceEntry#RADIO_BUTTON");
+
+            Click.clickAtNotVisible(
+                key_radioOption = "Include",
+                locator1 = "CommerceEntry#RADIO_BUTTON");
+        }
+        task ("And the user save the change clicking on save button") {
+            Button.clickSave();
+        }
```

Comment:
this should be part of the previous task, as you are just finishing the filter creation